### PR TITLE
Feature/dev

### PR DESCRIPTION
**Pull request by olegtaranenko**
build a proto app. Login page

### CODE REVIEW

File: proto-app/src/components/Auth/Form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+							backgroundImage: `url(${iconPass})`,
+						}}
+						value={passValue}
+						onChange={(evt) => {
+							setPassValue(evt.target.value);
+						}}
+					/>
+				</div>
+				<p className={classes.linkTerms}>
+					By logging in your agree to the &nbsp;
```
Comment (werzl) [inline proto-app/src/components/Auth/Form/Form.jsx:50]: spelling mistake: 
```suggestion
					By logging in you agree to the &nbsp;
```

Comment (werzl) [inline proto-app/src/components/Auth/Form/Form.jsx:50]: you can see there is too much space between this line and the terms of use link

```suggestion
					By logging in your agree to the 
```

File: proto-app/src/components/UI/Button/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import classes from "./Button.module.scss";
+import React from "react";
+
+const button = ({ onButtonClick, buttonId, btnType, children }) => (
```
Comment (werzl) [inline proto-app/src/components/UI/Button/Button.jsx:4]:
```suggestion
const button = ({ onButtonClick, buttonId, buttonType, children }) => (
```

Comment (werzl) [inline proto-app/src/components/UI/Button/Button.jsx:4]:
```suggestion
const Button = ({ onButtonClick, buttonId, btnType, children }) => (
```

File: proto-app/src/components/UI/Button/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+		onClick={onButtonClick}
+		className={classes.button}
+		type={btnType}
+		id={buttonId}
+	>
+		{children}
+	</button>
+);
+
+export default button;
```
Comment (werzl) [inline proto-app/src/components/UI/Button/Button.jsx:15]:
```suggestion
export default Button;
```

**Review by werzl — CHANGES_REQUESTED** [review]

File: proto-app/src/components/Auth/Form/Form.jsx (hunk 1 above)
Comment (olegtaranenko) [reply proto-app/src/components/Auth/Form/Form.jsx]: fixed

**Review by werzl — APPROVED** [review]

**Pull request merged**